charts/rasa-x/templates/rasa-deployments.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       labels:
         {{- include "rasa-x.selectorLabels" $ | nindent 8 }}
         app.kubernetes.io/component: {{ .serviceName }}
+      annotations:
+        checksum/values: {{ toYaml $.Values | sha256sum }}
```

Comment:
This seems like a brute force approach. Howabout only checking if the `rasa` part of the `Values.yml` changed? Same for the rasa-x change.

Reply:
Let's also use a helper template to avoid duplicating the logic